core/src/main/scala/org/apache/spark/storage/DiskBlockManager.scala

```diff
@@ -40,6 +46,7 @@ private[spark] class DiskBlockManager(conf: SparkConf, deleteFilesOnStop: Boolea
   /* Create one local directory for each path mentioned in spark.local.dir; then, inside this
    * directory, create multiple subdirectories that we will hash files into, in order to avoid
    * having really large inodes at the top level. */
+  private val configuredLocalDirs: Array[String] = Utils.getConfiguredLocalDirs(conf)
```

Comment:
Does the comment above is applied to this new line too? If not, could you move this new line on top of the comment?